src/store/podcasts.ts

```diff
@@ -22,8 +24,27 @@ export const usePodcastStore = defineStore("podcasts", {
       { id: "dev2", name: "dev 2 sandbox" },
       { id: "dev3", name: "dev 3 sandbox" },
     ],
+    readAccessPodcasts: [],
   }),
   getters: {
     getPodcasts: (state: State) => state.podcasts,
+    getReadAccessPodcasts: (state: State) => state.readAccessPodcasts,
+  },
+  actions: {
+    async getReadAccessPodcast() {
+      this.readAccessPodcasts = []
+      const db = getFirestore()
+      this.podcasts.map(async (el) => {
+        await getDoc(doc(collection(db, el.id)))
```

Comment:
mix of async/await and promises, not sure why?
could be simplified in a try/catch with async/await